Configuration/Services.yaml

```diff
@@ -49,3 +49,36 @@ services:
         iconIdentifier: 'content-widget-list'
         height: 'medium'
         width: 'medium'
+
+  dashboard.buttons.contact:
+    class: 'TYPO3\CMS\Dashboard\Widgets\Provider\ButtonProvider'
+    arguments:
+      $title: ''
+      $link: ''
+      $target: '_blank'
+
+  widgets.dashboard.widget.contact:
```

Comment:
Maybe we should not add a default widget for this as the default really is a not-useful wizard. Let's discuss this on Slack